cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -453,9 +459,11 @@ V1Deployment buildDeployment(RunId runid,
 
     V1ResourceRequirements resourceRequirements = new V1ResourceRequirements();
     Map<String, Quantity> quantityMap = new HashMap<>();
-    quantityMap.put("cpu", new Quantity(String.valueOf(vCores)));
+    float cpuMultiplier = Float.parseFloat(kubeMasterConf.getOrDefault(CPU_MULTIPLIER, DEFAULT_MULTIPLIER));
+    float memoryMultiplier = Float.parseFloat(kubeMasterConf.getOrDefault(MEMORY_MULTIPLIER, DEFAULT_MULTIPLIER));
+    quantityMap.put("cpu", new Quantity(String.valueOf((float) (vCores * cpuMultiplier))));
```

Comment:
how about using "milli" unit for CPU with rounding up like this?

quantityMap.put("cpu", new Quantity(String.format("%dm", (int)Math.ceil((float)vCores * 1000 * multiplier))

You can create a util function 
vCoresToMilliCpu(vcore, multplier) {
}